src/components/ItaliaTheme/Blocks/Listing/SimpleListTemplate.jsx

```diff
@@ -32,21 +31,12 @@ const SimpleListTemplate = ({
                 {items.map((item, index) => {
                   return (
                     <li>
-                      {item['@type'] === 'File' ? (
-                        <a
-                          href={flattenToAppURL(item.file.download)}
-                          title={item.file.filename}
-                        >
-                          {item.title}
-                        </a>
-                      ) : (
-                        <UniversalLink
-                          item={!isEditMode ? item : null}
-                          href={isEditMode ? '#' : null}
-                        >
-                          {item.title}
-                        </UniversalLink>
-                      )}
+                      <UniversalLink
```

Comment:
Consiglio l'utilizzo di ``ConditionalLink`` invece di ``UniversalLink``, così in edit non clicchiamo per sbaglio i link lavorando sul blocco

Reply:
Giusto